packages/rest-typings/src/v1/rooms.ts

```diff
@@ -792,6 +805,10 @@ export type RoomsEndpoints = {
 		}) => { message: IMessage | null };
 	};
 
+	'/v1/rooms.mediaEdit/:rid/:fileId': {
+		POST: (params: { fileName?: string }) => void;
```

Comment:
```suggestion
		POST: (params: { fileName: string }) => void;
```
fileName is required on the ajv definition.